README.md

```diff
@@ -141,6 +141,11 @@ installed on my machine - I can run Fish and it's VMX driver will automatically
 run VMX images. In case I have docker installed too - I can use both for different workloads or
 select the ones I actually want to use by `--drivers` option or via the API.
 
+In case you need to utilize multiple different configuration for the driver - you can use suffix
+for it's name: like `aws` and `aws/dev` - will effectively serve AWS workloads, but both will have
+different keys, so labels will be able to specify `driver: aws/dev` in order to use `aws/dev` config
+or `driver: aws` in order to use `aws` one - so will run in different AWS projects.
```

Comment:
The way this reads is a little confusing to me.  Suggestion:

```
In the event you need to use more than one configuration for a given driver, you can add
a suffix `/<name>`.  For example, `aws` and `aws/dev` will both use the AWS driver, but
use a different configuration.  In this example, labels created will need to specify either
`driver: aws` or `driver: aws/dev` to select which configuration to run.
```

Reply:
Yep, just changed and a bit updated README.md to reflect the current state.